### PR TITLE
all: disable interfacer in megacheck inside our CI script

### PR DESCRIPTION
**Pull request by weeellz**

### CODE REVIEW

File: .gometalinter.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+		"ineffassign",
+		"lll",
+		"maligned",
+		"megacheck",
+		"misspell",
+		"nakedret",
+		"safesql",
+		"structcheck",
+		"unconvert",
+		"unparam",
```
Comment (Quasilyte) [inline .gometalinter.json]: Voting for `unparam` removal.

File: .gometalinter.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		"errcheck",
+		"gas",
+		"goconst",
+		"gocyclo",
+		"goimports",
+		"golint",
+		"gotype",
+		"gotypex",
+		"ineffassign",
+		"lll",
```
Comment (Quasilyte) [inline .gometalinter.json]: Voting for `lll` removal.

File: .gometalinter.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+	"DisableAll": true,
+	"Enable": [
+		"deadcode",
+		"dupl",
+		"errcheck",
+		"gas",
```
Comment (Quasilyte) [inline .gometalinter.json]: `gas` is somewhat nasty, too.
It can stay for now.

Comment (weeellz) [reply .gometalinter.json]: Ok.

File: .gometalinter.json (hunk 2 above)
Comment (weeellz) [reply .gometalinter.json]: Removed.

File: .gometalinter.json (hunk 1 above)
Comment (weeellz) [reply .gometalinter.json]: Removed.

File: .gometalinter.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+		"maligned",
+		"megacheck",
+		"misspell",
+		"nakedret",
+		"safesql",
+		"structcheck",
+		"unconvert",
+		"varcheck",
+		"vet"
+  ]
```
Comment (Quasilyte) [inline .gometalinter.json]: Maybe this `]` can be aligned?
I think spaces should be replaced with tab there.
You can turn on whitespace display in your editor. Makes this issues obvious.

Comment (weeellz) [reply .gometalinter.json]: Done.

**Comment by Quasilyte** [comment]
Thanks!
Consider changing commit message when squashing.
LGTM.

**Pull request merged**